README.md

```diff
@@ -153,14 +153,14 @@ languagetool_rust = "^2.1"
 ### Example
 
 ```rust
-use languagetool_rust::{check::CheckRequest, server::ServerClient};
+use languagetool_rust::api::{check::CheckRequest, server::ServerClient};
```

Comment:
I just noticed that we have both `check::CheckRequest` and `words::add::Request`, which are two different naming conventions. I think it would be better to stick to one (the latter?).

Reply:
Sure, so you want it to look like `check::Request` and `check::Response`?